logging/monolog_email.rst

```diff
@@ -285,39 +286,43 @@ get logged on the server as well as the emails being sent:
     .. code-block:: php
 
         // config/packages/prod/monolog.php
-        $container->loadFromExtension('monolog', [
-            'handlers' => [
-                'main' => [
-                    'type'         => 'fingers_crossed',
-                    'action_level' => 'critical',
-                    'handler'      => 'grouped',
-                ],
-                'grouped' => [
-                    'type'    => 'group',
-                    'members' => ['streamed', 'deduplicated'],
-                ],
-                'streamed'  => [
-                    'type'  => 'stream',
-                    'path'  => '%kernel.logs_dir%/%kernel.environment%.log',
-                    'level' => 'debug',
-                ],
-                'deduplicated' => [
-                    'type'     => 'deduplication',
-                    'handler'  => 'symfony_mailer',
-                ],
-                'symfony_mailer' => [
-                    'type'         => 'symfony_mailer',
-                    'from_email'   => 'error@example.com',
-                    'to_email'     => 'error@example.com',
-                    // or a list of recipients
-                    // 'to_email'   => ['dev1@example.com', 'dev2@example.com', ...],
-                    'subject'      => 'An Error Occurred! %%message%%',
-                    'level'        => 'debug',
-                    'formatter'    => 'monolog.formatter.html',
-                    'content_type' => 'text/html',
-                ],
-            ],
-        ]);
+        use Symfony\Config\MonologConfig;
+
+        return static function (MonologConfig $monolog) {
+            $monolog->handler('main')
+                ->type('fingers_crossed')
+                ->actionLevel('critical')
+                ->handler('grouped')
+            ;
+
+            $monolog->handler('group')
+                ->members(['streamed', 'deduplicated'])
+            ;
+
+            $monolog->handler('streamed')
+                ->type('stream')
+                ->path('%kernel.logs_dir%/%kernel.environment%.log')
+                ->level('debug')
+            ;
+
+            $monolog->handler('deduplicated')
+                ->type('deduplicated')
+                ->handler('symfony_mailer')
+            ;
+
+            // still passed *all* logs, and still only logs error or higher
+            $monolog->handler('symfony_mailer')
+                ->type('symfony_mailer')
+                ->fromEmail('error@example.com')
+                ->toEmail(['error@example.com'])
+                // or a list of recipients
+                // ->toEmail(['dev1@example.com', 'dev2@example.com', ...])
```

Comment:
Im not sure we need to show that a "list of recipients" is possible since `toEmail()` requires an array.